src/bot/modules/stats.ts

```diff
@@ -167,7 +167,10 @@ export default class Stats {
         return { userId, points };
       })
       .sort((a, b) => b.points - a.points)
-      .map((entry) => `${userMention(entry.userId)} : ${entry.points} ${entry.points !== 1 ? 'points' : 'point'}`);
+      .map((entry) => {
+        const sep = entry.userId === interaction.user.id ? '**' : '';
+        return `${userMention(entry.userId)} : ${sep}${entry.points} ${entry.points !== 1 ? 'points' : 'point'}${sep}`;
+      });
```

Comment:
Sa serait pas mieux ?
```suggestion
      .map((entry) => {
        const sep = `${entry.points} ${entry.points === 1 ? 'point' : 'points'}`;
        return `${userMention(entry.userId)} : ${interaction.user.id === entry.userId ? bold(sep) : sep}`;
      });
```

Reply:
Ça ne serait pas mieux ?
```suggestion
      .map((entry) => {
        const points = `${entry.points} ${entry.points === 1 ? 'point' : 'points'}`;
        return `${userMention(entry.userId)} : ${interaction.user.id === entry.userId ? bold(points) : points}`;
      });
```